src/components/tooltip/Tooltip.tsx

```diff
@@ -126,10 +126,24 @@ class Tooltip extends React.Component<TooltipProps, State> {
         this.state = { isShown: !!props.isShown, hasRendered: false, wasClosedByUser: false };
     }
 
+    isControlled = () => {
```

Comment:
[nit] can this be in the original spot? order shouldnt matter in class component

Reply:
good point, updated